web/controllers/user_controller.ex

```diff
@@ -32,10 +23,11 @@ defmodule CoursePlanner.UserController do
 
     case Repo.update(changeset) do
       {:ok, user} ->
-        Users.notify_user(user, current_user, :user_modified, user_url(conn, :show, user))
+        user_url = user_show_url(user)
+        Users.notify_user(user, current_user, :user_modified, user_url)
```

Comment:
the notification needs the full url, but `user_show_url` generates only the path